### DISCUSSION TITLE
臺灣影史首創！《流麻溝十五號》AI聲音模擬技術 重現蔣經國原音

### DISCUSSION THREAD

**Discussion by Kotori0501** [1101-GS4719-C/Python-Community-GS4719-C#820] [General]
本偏文章是利用文字轉語音的AI技術，來模擬、合成蔣經國的聲音，還原其歷史性，給大眾觀看。

我覺得如果可以準確利用文字轉語音並合成自己想要的聲音，可以給對自己聲音不自信的人增添自信，因此是一項很好的技術。

https://tw.news.yahoo.com/%E8%87%BA%E7%81%A3%E5%BD%B1%E5%8F%B2%E9%A6%96%E5%89%B5-%E6%B5%81%E9%BA%BB%E6%BA%9D%E5%8D%81%E4%BA%94%E8%99%9F-ai%E8%81%B2%E9%9F%B3%E6%A8%A1%E6%93%AC%E6%8A%80%E8%A1%93-%E9%87%8D%E7%8F%BE%E8%94%A3%E7%B6%93%E5%9C%8B%E5%8E%9F%E9%9F%B3-124552097.html

**Comment by StevenWanglolz** [comment]
現在AI的技術可以拿來修復好多以前損毀的資料，太厲害了!

**Comment by CPCpyt** [comment]
雖然還原史實感覺很棒，但感覺有時候會有歷史幻想被破滅的瞬間XDD

**Comment by Doreen331** [comment]
之前學校所舉辦的某場週會也有邀請相關領域人士，在中央大學還原過世已久的鄧麗君女士，雖然之前就有耳聞這種技術，但實際發生在自己身邊還是會不禁讚嘆科技的進步！

**Comment by Evan0979** [comment]
要小心被有心人士配合deepfake使用,網路上的真真假假感覺越來越難分辨了!

**Comment by Floriaand** [comment]
不知道這項技術是不是早就被詐騙公司研發出來了，詐騙電話裡的聲音跟本人都有點像。

**Comment by Twenty0522** [comment]
雖然沒有聽過蔣經國的聲音，但因為這是一部募資的電影，又講述台灣的歷史，所以當時沒考慮影評如何，在上映的前幾天就去電影院看了。我覺得這對ai技術的宣傳與應用是非常好的合作。

**Comment by Valentinaa0214** [comment]
這部電影我有去看，當時心想這也太像了吧哈哈哈，現在才知道原來是AI模擬生成的！
經過之前的換臉事件，希望這種聲音技術不要被有心人士不當利用才好！

**Comment by woohsin** [comment]
我們也應該注意到這種技術可能帶來的一些潛在問題，比如認知權利問題、個人身份被冒用等。在使用這樣的技術時，我們應該要謹慎對待，並確保遵守相關的法律法規和道德準則。

總的來說，這項技術確實具有一定的應用價值，但同時也需要我們慎重考慮其可能帶來的各種影響，並提出相應的管理和監管措施。

**Comment by sandyho0307** [comment]
我覺得利用AI技術重現名人的聲音，提供了一種新的文化保存方式，使得歷史人物和事件能以全新的方式呈現給現代觀眾。然而，這種技術可能也會帶來倫理和道德上的問題。

**Comment by tshsu04224** [comment]
之前看過一個節目舞台是還原鄧麗君的聲音和表演者一起合唱，我覺得這種技術的進步讓我們可以重見過往，將創新融入歷史，是一個不錯的科技發展方向，但若有心人士要利用他來做詐騙等行為，未來可能就更加難分辨真偽，所以也要嚴加防範科技帶來的危害。

**Comment by cassy0201** [comment]
感覺這項技術可以重現我們思念的人的聲音，但有心人士可能也會利用這點來進行犯罪，必須小心防範

**Comment by NcuJerryKao** [comment]
對於利用文字轉語音的AI技術來合成特定聲音的應用，確實有許多潛在的應用價值。不僅可以用來模擬歷史性人物的聲音，也可以幫助那些對自己的聲音不自信的人增加自信。此外，這項技術還可以應用在語音助手、有聲書、語音導覽等領域，為人們提供更多元化的體驗。
然而，在應用這項技術時，也需要注意一些倫理與隱私問題，例如未經授權使用他人聲音、語音合成造成誤導等問題。因此，在進行這項技術應用時，應該謹慎考慮，確保合法合理使用，並尊重他人的權利與隱私。
總的來說，AI技術的發展為我們帶來了許多機遇與挑戰，我們應該善加利用這項技術，同時也要注意其影響，確保其應用符合道德與法律標準。希望這項技術能夠為人們帶來更多便利與樂趣。
